pkg/app/server.go

```diff
@@ -371,3 +395,12 @@ func buildMetricsServer(m *metricshandler.MetricsHandler, durationObserver prome
 	})
 	return mux
 }
+
+func md5HashAsMetricValue(data []byte) float64 {
+	sum := md5.Sum(data) //nolint:gosec
```

Comment:
Just wondering, why is the linter not happy with this?

Reply:
md5 is considered a weak cryptographic hash function and gosec has the module on a warn list. Usage here is fine though, as we're checking for file contents and are not hashing passwords or similar.

Reply:
md5 is cryptographically broken, so the linter doesn't like uses of it. But for checksumming it should be fine.